apps/desktop-wallet/src/pages/UnlockedWallet/OverviewPage/AddressesContactsList.tsx

```diff
@@ -78,7 +78,7 @@ const AddressesContactsList = ({ className, maxHeightInPx }: AddressesContactsLi
 const AddressesList = ({ className, isExpanded, onExpand, onAddressClick }: AddressListProps) => {
   const addresses = useAppSelector(selectAllAddresses)
   const fiatCurrency = useAppSelector((s) => s.settings.fiatCurrency)
-  const { data: price } = useGetPriceQuery(currencies[fiatCurrency].ticker)
+  const price = useAppSelector(selectAlphPrice)
```

Comment:
Let's rename this `alphPrice` to avoid confusion (similar to how you did in the following file)?